gnoboard/src/screens/devmode/index.tsx

```diff
@@ -35,12 +37,18 @@ function DevMode() {
     setAppConsole('Loading account...');
 
     try {
-      const response = await listKeyInfo();
-      console.log(response);
+      const port = await getTcpPort();
+
+      console.log('port: ', port);
+      const client = await createClient(port);
```

Comment:
You also call `createClient` in `useGno`. I don't think you should call `createClient` in two places. Can you put `useGno` here?